staff/aleix-caminal/cv/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+    <head>
+        <meta charset="utf-8">
+        <title>Curriculum Vitae</title>
+    </head>
+    <body>
+        <main>
+            <header>
+                <aside class="image">
+                    <img src="assets/img/profile.jpg" alt="profile">
+                </aside>
+                <aside class="info">
+                    <h4 class="name">Charles Tucker III</h4>
+                    <p class="email">E-mail: <a href="mailto:charlestrip@starfleet.com">charlestrip@starfleet.com</a></p>
+                    <p class="mobile">Mobile: 07791 858292</p>
+                    <address>
```

Comment:
I saw that you did not include the person name, email and mobile phone to the `<address>` tag. As fas as I am concerned `<address>` tag also have the resposability for thous fields.

Further information:
[MDN Adress tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address)

Reply:
good point